app/assets/scripts/components/slate/plugins/equation/equation-element.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback } from 'react';
+import T from 'prop-types';
+import styled from 'styled-components';
+import { Node, Transforms } from 'slate';
+import { ReactEditor, useSelected, useSlate } from 'slate-react';
+import { BlockMath } from 'react-katex';
+
+import { themeVal } from '@devseed-ui/theme-provider';
+
+const PreviewBody = styled.div`
+  cursor: pointer;
+  border: 1px solid;
+  border-color: ${({ inFocus }) =>
+    inFocus ? themeVal('color.baseAlphaE') : 'transparent'};
+  border-radius: ${themeVal('shape.rounded')};
+
+  &:hover {
+    border-color: ${themeVal('color.baseAlphaE')};
+  }
+`;
```

Comment:
The hover and selected color should be based on `primary` and the selected be darker.
It would also be good to have a smooth transition like other block elements (ex table) do.